src/utils.ts

```diff
@@ -1,6 +1,6 @@
 import c from "cookie";
 import { ServerResponse } from "http";
-import { Options, SessionData } from "./types";
+import { Options, Session, SessionData } from './types'
```

Comment:
Could you respect the current formatting style?

Reply:
Sorry, I fixed.